fv/certs/admin-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Github might flag a security warning on our repo for checking in private keys?

Reply:
We're actually already doing this for the FV tests: https://github.com/projectcalico/felix/blob/master/fv/private.key

I just added a few more. We can probably remove the old one, though. 

And of course this doesn't change the fact that GH might warn about it in the future, though....